PolyEngine/Engine/Src/Rendering/Particles/ParticleEmitter.hpp

```diff
@@ -48,7 +48,7 @@ namespace Poly
 			int BurstSizeMax = 5;
 			eSimulationSpace SimulationSpace = eSimulationSpace::WORLD_SPACE;
 			// TODO: move to material
-			SpritesheetSettings SprsheetSettings;
+			SpritesheetSettings Spritesheet;
```

Comment:
This actually is quite misleading variable name. Maybe `Settings` is enough.

Reply:
Our current CamelCode code style makes difficult to create members with the same name as class name. Same issue is with `WorldData` in `SceneView`. There `Settings` would be ambiguous for me.